charts/zqd/values.yaml

```
@@ -0,0 +1,87 @@
+# Default values for zqd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+datadir: "/spaces"
```

Because this can also be an s3 uri perhaps change this to:
```suggestion
datauri: "file:///spaces"
```